packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/steps/component/DecryptedResult.tsx

```diff
@@ -6,20 +6,20 @@ import React from "react";
 import { SectionTitle } from "../../../common/stepStyles";
 import { FormContent } from "../../../common/type/WizardType";
 import {
-  PublicodesContextInterface,
+  PublicodesPreavisRetraiteResult,
   PublicodesResult,
+  usePublicodes,
 } from "../../../publicodes";
 
 type Props = {
   data: FormContent;
-  publicodesContext: PublicodesContextInterface;
 };
 
 const ShowResult: React.FC<{
   result: PublicodesResult;
   agreementMaximumResult: PublicodesResult | null;
 }> = ({ result, agreementMaximumResult }) => {
-  if (result.value > 0) {
+  if (result?.value && result.value > 0) {
```

Comment:
je crois qu'on peut mettre directement :

```suggestion
 if (result?.value > 0) {
```

Reply:
En effet, c'est juste le AgreementResult qui peut être `null`. Je pensais tout normalisé, or le legal result n'est jamais null